be/src/storage/rowset/segment_iterator.cpp

```diff
@@ -337,11 +353,11 @@ Status SegmentIterator::_init() {
     // Use indexes and predicates to filter some data page
     RETURN_IF_ERROR(_init_bitmap_index_iterators());
     RETURN_IF_ERROR(_get_row_ranges_by_keys());
+    RETURN_IF_ERROR(_get_row_ranges_by_rowid_range());
```

Comment:
Why change the order?

Reply:
By using `_get_row_ranges_by_rowid_range`, there will be less rowids, and make the following index operations more quickly.